Libraries/CustomComponents/NavigationExperimental/NavigationCardStack.js

```diff
@@ -221,16 +220,7 @@ class NavigationCardStack extends React.Component<DefaultProps, Props, void> {
   _configureTransition = () => {
     const isVertical = this.props.direction === 'vertical';
     const animationConfig = {};
-    if (
-      !!NativeAnimatedModule
-
-      // Gestures do not work with the current iteration of native animation
-      // driving. When gestures are disabled, we can drive natively.
-      && !this.props.enableGestures
-
-      // Native animation support also depends on the transforms used:
-      && NavigationCardStackStyleInterpolator.canUseNativeDriver(isVertical)
-    ) {
+    if (NavigationCardStackStyleInterpolator.canUseNativeDriver(isVertical)) {
```

Comment:
1. Yea I think showing the warning is fine, it will be shown from TouchableOpacity too anyway so it doesn't really matter.

2. I guess we could remove it. @ericvicenti any reason why this was added in the first place?